test/source/tests/tooling/index.ts

```diff
@@ -1,14 +1,18 @@
 /* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
 
-import * as ava from 'ava';
+import { ExecutionContext } from 'ava';
+import { TestUrls } from '../../browser/test-urls';
 
 import { Consts } from '../../test';
 
-export type AvaContext = ava.ExecutionContext<unknown> & {
+export type AvaContext = ExecutionContext<unknown> & {
   retry?: true;
   attemptNumber?: number;
   totalAttempts?: number;
   attemptText?: string;
+  extensionDir?: string;
+  urls?: TestUrls;
+  closeMockApi?: () => Promise<void>;
```

Comment:
Maybe `closeMockApi` wouldn't have to be on `AvaContext` if it was in try/finally above